cmd/argocd/commands/admin/backup.go

```diff
@@ -139,7 +150,9 @@ func NewImportCommand() *cobra.Command {
 		verbose                  bool
 		stopOperation            bool
 		ignoreTracking           bool
+		retryOnConflict          bool
```

Comment:
Should be overrideOnConflict instead to clarify the actual behavior, or override/force if we always override the resource without checking for conflict first. I would tend to prefer overrideOnConflict, so conflicts can be logged to increase visibility.

Reply:
agreed on the name, change it to `overrideOnConflict`